rotkehlchen/tasks/manager.py

```diff
@@ -500,6 +504,31 @@ def _maybe_update_snapshot_balances(self) -> None:
                     ignore_cache=True,
                 )
 
+    def _maybe_update_curve_pools(self) -> None:
+        curve_caches = GlobalDBHandler().get_request_caches(
+            cache_type=RequestsCacheType.CURVE_POOL_ADDR,
+        )
+        if len(curve_caches) == 0:
+            schedule = True
+        else:
+            last_update_ts = GlobalDBHandler().get_requests_cache_last_queried_ts(
+                cache_type=RequestsCacheType.CURVE_POOL_ADDR,
+                name=curve_caches[0],
+            )
+            assert last_update_ts is not None  # should not be None
+            if ts_now() - last_update_ts >= CURVE_POOLS_UPDATE_SECS:
+                schedule = True
+            else:
+                schedule = False
```

Comment:
Can you reverse this if (switch the code in the else part by the one in the if part), because is the one that will be running most of the cases